go/libkb/output_windows.go

```diff
@@ -92,6 +92,12 @@ type ColorWriter struct {
 	mutex *sync.Mutex
 }
 
+// Fd returns the underlying file descriptor. This is mainly to
```

Comment:
Looks like stuff still tries to write color codes other than the logs, so we should leave the ColorWriter plumbing in there anyway, I think.